edb/server/protocol/auth_ext/http.py

```diff
@@ -185,11 +185,12 @@ def _make_session_token(self, identity_id: str) -> str:
         expires_in = auth_expiration_time.to_timedelta()
         expires_at = datetime.datetime.utcnow() + expires_in
 
-        claims = {
+        claims: dict[str, Any] = {
             "iss": self.base_path,
             "sub": identity_id,
-            "exp": expires_at.astimezone().timestamp(),
         }
+        if expires_in.total_seconds() != 0:
+            claims["exp"] = expires_at.astimezone().timestamp()
```

Comment:
I'm not sure what the easiest way to add a test case for this is since we have the extension setup in the test class itself and I don't want to lose testing the non-zero case. Any suggestions?